.github/ISSUE_TEMPLATE/specreview_template.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Specification Review Issue
+about: Start a request for a specification creation/plan/release... lifecycle review
+title: \[MP Component] \[Version] Specification Review
+labels: creation,plan,progress,release
+assignees: ''
+---
+
+## Specification issue template
+When creating a specification project lifecycle review, create issues in the [MicroProfile-WG repository](https://github.com/microprofile/microprofile-wg) with the content defined as follows.
+
+- [ ] Specification name and version (two digit):
+- [ ] Add a label from one of: `creation`, `plan`, `progress`, `release`
+- [ ] The URL of the OSSRH staging repository for the spec (using the staging repository id):
+      (eg, https://oss.sonatype.org/content/repositories/orgeclipsemicroprofile-1357/org/eclipse/microprofile/config/microprofile-config-spec/2.0-RC9/)
+ - [ ] Specification PDF in the form of microprofile-_project_-spec-x.y.pdf where _project_ is the microprofile specification short project name (config, health, ...)
```

Comment:
`microprofile-_project_-spec-x.y-pdf.pdf` as we have to add classifier if we want to push both pdf and html to maven central.

Reply:
Really?  The type of the file is not sufficient to differentiate (.pdf, .html)?  That's kind of ugly...

Reply:
That can't be right...  We already have examples of the same file name being used with different extensions (.jar and .pom):  https://oss.sonatype.org/content/repositories/orgeclipsemicroprofile-1357/org/eclipse/microprofile/config/microprofile-config-tck/2.0-RC9/

Reply:
No worries, I was able to remove the classifier name.